luxonis_train/utils/boundingbox.py

```diff
@@ -411,6 +669,182 @@ def non_max_suppression(
     return output
 
 
+def non_max_suppression_obb(
```

Comment:
This is almost identical to `non_max_suppression`. Can you improve `non_max_suppression` to also support oriented bboxes instead of having this function?